docs/blog/2019-03-22-introducing-gatsby-preview-beta/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Introducing Gatsby Preview Beta
+date: 2019-03-22
+author: Marisa Morby
+excerpt: "Introducing the Gatsby Preview Beta: learn more and sign up today for early access to our first commerical product."
+tags:
+  - preview
+  - cloud
+  - commercial
+---
+
+## What we’ve been up to
+
+Back in July we were excited to send out our [alpha version of Gatsby Preview](https://www.gatsbyjs.org/blog/2018-07-17-announcing-gatsby-preview/). The team has been working hard getting your feedback and incorporating it into Gatsby Preview beta.
+
+We’re excited to let you know that this week, Gatsby Preview beta will be officially opening!
+
+_In beta, we’ll be supporting Gatsby sites that are set up with Contentful._
+
+## What Gatsby Preview can do for you and your team
+
+In a traditional CMS setup, content creators and editors can preview their changes before they go live. Without Gatsby Preview, teams using Gatsby needed to set up their own preview infrastructure or do without. This was a problem because content teams need to see what changes they’re making as it’s happening. And assuming teams had time to create their own preview infrastructure, it was still a time-consuming process. We wanted to solve this for teams using Gatsby.
+
+With Gatsby Preview beta, you can connect your GitHub repositories and Contentful CMS to our service, and get a live URL where content creators can see their changes in context before going live. Want to check on your new article title, or look at the changes you made to your post? Gatsby Preview lets you see it!
+
+## What’s new with Gatsby Preview beta
+
+For the beta version of Gatsby Preview, we focused on adding a UI over the existing functionality, made the initial set-up less manual, and added ways for your team to get involved. In beta, we’ll only be supporting Gatsby sites managed through the Contentful CMS with support for other CMS’s coming soon.
```

Comment:
```suggestion
For the beta version of Gatsby Preview, we focused on adding a UI over the existing functionality, made the initial set-up less manual, and added ways for your team to get involved. In beta, we’ll only be supporting Gatsby sites managed through the Contentful CMS with support for other CMS platforms coming soon.
```
A suggestion since CMS's is a little awkward.